.stylelintrc.json

```diff
@@ -2,7 +2,8 @@
 	"rules": {
 		"color-no-hex": true,
 		"color-named": "never",
-		"declaration-block-no-duplicate-properties": true,
+    "declaration-block-no-duplicate-properties": true,
```

Comment:
It's a nit, but indentation doesn't match here (4 spaces instead of 2 tabs). Edits should follow whatever conventions are already established in the file.

Reply:
My apologies.  I opened up a follow up PR to fix this issue: https://github.com/internetarchive/openlibrary/pull/1299.